src/io/input/xml/XMLFileReader.cpp

```diff
@@ -3,26 +3,85 @@
 #include <optional>
 
 #include "io/logger/Logger.h"
+#include "io/particle_spawners/CuboidSpawner.h"
 
-SimulationParams XMLFileReader::readConfiguration(const std::string& filepath, ParticleContainer& particle_container) const {
-    if (filepath.empty()) {
-        Logger::logger->error("No input file specified.");
-        exit(-1);
-    }
-
+SimulationParams XMLFileReader::readFile(const std::string& filepath, ParticleContainer& particle_container) const {
     try {
         std::unique_ptr< ::configuration> config = configuration_(filepath);
 
-        Logger::logger->info("Successfully parsed XML file.");
+        Logger::logger->info("Successfully parsed XML file.Simulation parameters: ");
         Logger::logger->info("FPS: {}", config->fps().get());
```

Comment:
Logging all supplied values should only happen in the main function, not deeply nested inside some function. 

Since there all the information gets combined. Its easier to get an overview of all parameters and statistics